views/repo_list.pug

```diff
@@ -1,9 +1,13 @@
 h1
-  Github Repositories
+  Github List Repositories
 
 .git-repo-areas
 
   .git-repo( ng-repeat="repo in globalState.user.repos()" ng-controller="ListItemCtrl" )
     .repo-name
-      span( ng-bind="repo.full_name" )
-      a.link-out( ng-href="{{repo.html_url}}" target="_blank" )
+    <h3 class="titlerepo">{{repo.name}}</h3>
+    <h5 class="subtitlerepo"><span>Forked from </span>{{repo.full_name}}</h5>
+    <h4 class="descriptionrepo">{{repo.description}}</h4>
+    <h4 class="totalforkrepo"><span class="fa fa-code-fork"></span>{{repo.forks_count}} <span class="updaterepo">Updated on {{repo.updated_at | date : short : timezone}}</span></h4>
+    <a ng-href="{{repo.html_url}}" class="btn btn-red"><span class="fa fa-github"></span> View Repo</a>
+    <div class="linerepo"></div>
```

Comment:
why not apply border-bottom directly on the `.git-repo` element instead of creating an additional element just to render the border bottom